packages/flex-plugin-scripts/src/scripts/deploy.ts

```diff
@@ -46,6 +47,23 @@ export const _verifyPath = (baseUrl: string, build: Build) => {
   return existingAssets.every(checkPathIsUnused) && existingFunctions.every(checkPathIsUnused);
 };
 
+/**
+ * Validates Flex UI version requirement
+ * @param flexUI      the flex ui version
+ * @param allowReact  whether this deploy supports unbundled React
+ * @private
+ */
+export const _verifyFlexUIVersion = (flexUI: string, allowReact: boolean) => {
+  const coerced = semver.coerce(flexUI);
+  const UISupports = semver.satisfies('1.19.0', flexUI) || (coerced && semver.satisfies(coerced, '>=1.19.0'));
+  if (allowReact && !UISupports) {
+    throw new FlexPluginError(singleLineString(
+      `The ui_version ${flexUI} is incompatible with unbundled React.`,
```

Comment:
"Your Flex account's flex-ui version is currently set to use ${flexUI}, which is incompatible with unbundled React.  Please upgrade your version to >= 1.19.0 before deploying this plugin.